sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/Application.java

```diff
@@ -1,14 +1,20 @@
 package com.sap.ai.sdk.app;
 
+import com.sap.ai.sdk.core.AiCoreService;
+import com.sap.cloud.sdk.services.openapi.apiclient.ApiClient;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.web.servlet.ServletComponentScan;
 import org.springframework.context.annotation.ComponentScan;
 
+/** Main class to start the Spring Boot application. */
 @SpringBootApplication
 @ComponentScan({"com.sap.cloud.sdk", "com.sap.ai.sdk.app"})
 @ServletComponentScan({"com.sap.cloud.sdk", "com.sap.ai.sdk.app"})
-class Application {
+public class Application {
+  /** The API client connected to the AI Core service. */
+  public static final ApiClient API_CLIENT = new AiCoreService().client();
```

Comment:
(Minor)

I'm not a fan of this implementation/declaration as it's _unusual_.

More common approach would be to define a `@Bean` and `@Autowired` it upon usage in controller classesn.

```java

@Bean
public ApiClient getApiCoreClient() {
  return new AiCoreService().client();
}
```

But we can also do this in a separate PR. That'll be fine.